### ISSUE TITLE
Using replace not able to use style

### ISSUE THREAD

**Issue by on2air** [remarkablemark/html-react-parser#172]
I'm using following replace to get notified when an image is loaded:
```
replace: function(domNode) {
            if(domNode.name === 'img'){
                console.log('replacing img', domNode)
                return (
                    <img
                        {
                            ...domNode.attribs
                        }
                        onLoad={() => console.log('image loaded')}
                    />
                )
            }

        }
```

If the image has a style attribute, when i add it back in via the domNode.attribs it throws an error since its not an style object as expected for the jsx img.  its still a string.  How can I convert the attributes to be jsx ready to passthru in the replace?

**Comment by cbbfcd** [comment]
use `attributesToProps`

**Comment by on2air** [comment]
ok, thanks.  I also got it working this way by just adding to the domNode.attribs and then passing to domToReact

`
domNode.attribs['onLoad'] = this.imgLoad
return domToReact(domNode)
`

**Comment by remarkablemark** [comment]
Published [0.14.0](https://github.com/remarkablemark/html-react-parser/releases/tag/v0.14.0):

[npm](https://www.npmjs.com/package/html-react-parser):

```sh
npm install html-react-parser@0.14.0
```

[yarn](https://yarnpkg.com/package/html-react-parser):

```
yarn add html-react-parser@0.14.0
```
